static/app/components/events/interfaces/utils.tsx

```diff
@@ -125,6 +125,10 @@ export function getCurlCommand(data: EntryRequest['data']) {
     result += ' \\\n -X ' + data.method;
   }
 
+  // Compare to null using != rather than !== to also catch undefined.
+  // eslint-disable-next-line no-eq-null, eqeqeq
+  data.headers = data.headers?.filter(h => h != null);
```

Comment:
we have a defined which also does typescript type narrowing (doesn't really matter in this case) https://github.com/getsentry/sentry/blob/master/static/app/utils.tsx#L123
```suggestion
  data.headers = data.headers?.filter(defined);
```

Reply:
Oh, good call. Fixed.